backend/app/categories/controller.py

```diff
@@ -17,9 +18,23 @@
 async def get_category_route(category_id: str,
                              db: AsyncIOMotorClient = Depends(get_database)) -> ItemsInResponse:
     """Get the items under a given category"""
-    # TODO add fetching from category
-    return ItemsInResponse()
-
+        
+    if category_id not in CategoryEnum.__members__:
+        return ItemsInResponse(success=False, error=[f"Invalid category {category_id}"])
+    
+    loader = None
+    if category_id == CategoryEnum.movies.value:
+        loader = Movie
+    if category_id == CategoryEnum.anime.value:
```

Comment:
Also all the other conditions should be with `elif` not just if, otherwise irrespective of the condition satisfied, it will try to check each condition.
```python
    if category_id == CategoryEnum.music:
        loader = Music
    elif category_id == CategoryEnum.shows:
        loader = Show
    elif category_id == CategoryEnum.books:
        loader = Book
    # similarly for the others
```